pkg/apis/certificates/types.go

```diff
@@ -42,6 +42,20 @@ type CertificateSigningRequestSpec struct {
 	// Base64-encoded PKCS#10 CSR data
 	Request []byte
 
+	// Requested signer for the request. It is a qualified name in the form:
+	// `scope-hostname.io/name`.
+	// If empty, it will be defaulted for v1beta1:
+	//  1. If it's a kubelet client certificate, it is assigned
```

Comment:
we also now default values for kubelet serving.  I think we can shorten this by removing the information about how we make the determination.